PixivModelFanbox.py

```diff
@@ -255,8 +310,7 @@ def parseBody(self, jsPost):
                 elif block["type"] == "embed":  # Implement #470
                     embedId = block["embedId"]
                     if embedId in jsPost["body"]["embedMap"]:
-                        embedStr = self.getEmbedData(jsPost["body"]["embedMap"][embedId], jsPost)
-                        self.body_text = f"{self.body_text}<br />{embedStr}"
+                        self.body_text += self.getEmbedData(jsPost["body"]["embedMap"][embedId], jsPost)
```

Comment:
Sequence `embed` block will wrap to one line, when the author link plural posts at once.

Reply:
hmm. how about we just wrap it up with a `p` tag. everything would be wrapped in `p`. i think that would be neat.

Reply:
i will do some tests....

Reply:
hmm, just tested with posting a post.
1. for images and files, during editing, even if i deleted the empty line between 2 continuous files or images or one of each, after publish, there is still a blank `p` between them.
2. not the same with embed, no empty `p`s between or before them.

i think i will just wrap them in `p` tags, if you don't mind?

Reply:
sorry, earlier reply was wrong. please ignore.
just tested with posting a post.
1. for images, during editing, even if i deleted the empty line between 2 continuous images, after publish, there seem always to be a blank p between and after them.
2. not the same with embeds or files, no empty ps between or before them.

i think i will just wrap them in p tags, if you don't mind?